skills/src/lasr_skills/look_at_person.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+import smach_ros
+from geometry_msgs.msg import Point, PointStamped
+import smach
+from vision import Get3DImage
+from lasr_vision_msgs.srv import BodyPixDetection, BodyPixDetectionRequest
+from lasr_vision_msgs.msg import BodyPixMaskRequest
+from lasr_skills import LookToPoint
+import cv2
+import rospy
+from sensor_msgs.msg import Image
+from cv2_img import cv2_img_to_msg, msg_to_cv2_img
+from markers import create_and_publish_marker
+from visualization_msgs.msg import Marker
+from cv2_pcl import pcl_to_img_msg
+import ros_numpy as rnp
+
+DEBUG = True
+
+class CheckEyes(smach.State):
```

Comment:
A bit picky, but I think we should nest this inside of the `LookAtPerson` class (as it can't really be used on its own!)